config/ola.m4

```diff
@@ -25,10 +25,10 @@ AC_REQUIRE_CPP()
 PKG_CHECK_MODULES(libprotobuf, [protobuf >= $1])
 
 AC_MSG_CHECKING([protobuf library version])
-PROTOBUF_VERSION=`pkg-config --modversion protobuf`;
+PROTOBUF_VERSION=`protoc --version | awk '{print [$]2}'`;
+AC_MSG_NOTICE([PROTOBUF VERSION $PROTOBUF_VERSION])
```

Comment:
I'm not hugely keen on adding another means of processing into the mix and we've already got a load of Protobuf versioning stuff in via https://github.com/OpenLightingProject/ola/commit/d4414feefd95adf9c91d7eaf1e94380296c35f7a so hopefully we can just reuse some of that to make a Java version that's compatible.

Reply:
As suggested, took it out of the PR with commit e792f5255f4ddf1a35be0cef7e1cb1ab16f0c853